src/Components/Authentication/Footer.tsx

```diff
@@ -16,21 +18,37 @@ export const Footer = (props: FooterProps) => {
     handleTypeChange,
     inline,
     mode,
+    onAppleLogin,
     onFacebookLogin,
     showRecaptchaDisclaimer,
   } = props
 
   switch (mode) {
     case "login": {
+      const thirdPartyLogin = sd.ENABLE_SIGN_IN_WITH_APPLE ? (
+        <FooterText>
+          {"Login using "}
```

Comment:
I think I need the bracket because of the trailing space, correct me if I am wrong or there is a prettier way to do this. But "Login" should be "Log in" so glad this line got more attention either way!